### PR TITLE
add pytest-redis

### PR DESCRIPTION
**Pull request by woutdenolf**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pytest-redis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - pytest_redis
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ClearcodeHQ/pytest-redis
+  summary: Redis fixtures and fixture factories for Pytest.
+  license: LGPL-3.0-only
```
Comment (dopplershift) [inline recipes/pytest-redis/meta.yaml:39]: Looks like it's [actually](https://github.com/ClearcodeHQ/pytest-redis/blob/c020d6b7d7f78da117f67252dc43f6f4755dc112/src/pytest_redis/__init__.py#L9-L10):
```suggestion
  license: LGPL-3.0-or-later
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a few tweaks around the license and this is good to go.

File: recipes/pytest-redis/COPYING [hunk 2]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING:1]: Thanks for the review! Do you mean I should remove the `license_file:` section from the recipe? And remove the license files as well?

[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING:1]: Or just remove the files and keep the `license_file:` section?

[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING]: I assume you meant remove the files and keep the `license_file: section`

[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING]: done

File: recipes/pytest-redis/COPYING.lesser [hunk 3]
```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```
[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING.lesser]: I assume you meant remove the files and keep the `license_file: section`

[the remark this answers is not in the document]
Comment (woutdenolf) [reply recipes/pytest-redis/COPYING.lesser]: done

File: recipes/pytest-redis/COPYING (hunk 2 above)
[the remark this answers is not in the document]
Comment (dopplershift) [reply recipes/pytest-redis/COPYING]: That's correct. Thanks!

**Review by dopplershift — APPROVED** [review]

**Pull request merged**